src/sentry/static/sentry/app/constants/index.tsx

```diff
@@ -19,13 +19,14 @@ export const API_SCOPES = [
   'org:read',
   'org:write',
   'org:admin',
+  'org:integrations',
```

Comment:
It seems we omitted this scope for API tokens in the past. This change will allow users to make user tokens with this scope.

Reply:
Do the endpoints also allow that scope to be used?

Reply:
@markstory Yea, we are definitely using `org:integrations` in certain scope maps.